.travis/install_saw.sh

```diff
@@ -32,7 +32,7 @@ mkdir -p "$DOWNLOAD_DIR"
 cd "$DOWNLOAD_DIR"
 
 #download saw binaries
-curl --retry 3 https://s3-us-west-2.amazonaws.com/s2n-public-test-dependencies/saw-0.2-2017-07-27-Ubuntu14.04-64.tar.gz --output saw.tar.gz;
+curl --retry 3 https://saw.galois.com/builds/nightly/saw-0.2-2019-03-08-Ubuntu14.04-64.tar.gz --output saw.tar.gz;
```

Comment:
Is it guaranteed that this file will always be there?  We've had issues in the past with old SAW builds disappearing.

Reply:
This link should now work: `https://s3-us-west-2.amazonaws.com/s2n-public-test-dependencies/saw-0.2-2019-03-08-Ubuntu14.04-64.tar.gz`